webservice/model/assignment.go

```diff
@@ -174,9 +187,11 @@ func (repo *AssignmentRepository) Insert(assignment Assignment) (int, error) {
 		// Create query string
 		query := "UPDATE assignments SET review_id = ? WHERE id = ?;"
 		// Prepare and execute query
-		_, err := db.GetDB().Exec(query, assignment.ReviewID, id)
+		_, err := tx.Exec(query, assignment.ReviewID, id)
 		// Check for error
 		if err != nil {
+			log.Fatal(err.Error())
```

Comment:
No need for logging inside here, cuase error is a return value.
And don't use log.Fatal() cauase it will stop the whole application.